src/org/typo3/chefci/helpers/Slack.groovy

```diff
@@ -0,0 +1,42 @@
+package org.typo3.chefci.helpers
+
+class Slack implements Serializable {
+    def script
+
+    JenkinsHelper jenkinsHelper
+
+    Slack(script) {
+        this.script = script
+        jenkinsHelper = new JenkinsHelper(script)
```

Comment:
your feedback is important! having couple of helper things certainly makes it hard to oversee..